test-files/cases/network/conflicts/highway-012/Expected.osm

```diff
@@ -1,41 +1,41 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
     <bounds minlat="-1.78022160047" minlon="-3.58724003701" maxlat="-1.773242914599999" maxlon="-3.58351785726"/>
-    <node visible="true" id="-767" timestamp="1970-01-01T00:00:00Z" version="1" lat="-1.7789556541828464" lon="-3.5855309496874908">
+    <node visible="true" id="-739" timestamp="1970-01-01T00:00:00Z" version="1" lat="-1.7789556541828464" lon="-3.5855309496874908">
```

Comment:
Outputs with lower numbers mean that new nodes/ways/relations are created less often and modified more.  This means that the way joiner is able to run better also.

Reply:
good deal.